cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8080",
+  "projectId": "pk2d9c"
```

Comment:
Shouldn't this be an env variable too?

Reply:
It can be [either](https://docs.cypress.io/guides/dashboard/projects.html#Project-ID) way. Though the cypress dashboard is public so making this env var won't make a difference. Also, it's generated automatically by the cypress command to identify all the runs across different machines, if for some reason it changed I think it's good to break the setup for all users.